docs/developer-docs/defi/tokens/asset_flow/index.mdx

```diff
@@ -0,0 +1,139 @@
+---
+keywords: [intermediate, tokens, tutorial, ledger, asset flow]
+---
+
+import TabItem from "@theme/TabItem";
+import { AdornedTabs } from "/src/components/Tabs/AdornedTabs";
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+import { GlossaryTooltip } from "/src/components/Tooltip/GlossaryTooltip";
+
+
+# Asset Flow
+
+<MarkdownChipRow labels={["Intermediate" , "Tutorial"]} />
+
+## Overview
+This guide will show you how to conduct different asset flow operations for both, the ICP ledger and ICRC-1 compliant ledgers.
+For reference on what an ICP and ICRC-1 ledgers are, please read the [documentation](/docs/developer-docs/defi/tokens/ledger/overview.mdx) on them. 
+For reference on the different standards for these ledger, you can find the relevant documentation [here](/docs/developer-docs/defi/tokens/token-standards.mdx). 
+* Transfer: A simple transfer between two entities.
+* Approve: Approving tokens to be spent by another enitty. 
+* Mint: Creating new tokens (**Requires Minting Account priviledges**) 
+* Burn: Destroying tokens. 
+* TransferFrom: Spend tokens on behalf of a previously approved entity. 
+
+The asset flow methods for both ledger types will be the same. 
+There are multiple ways on how to create the corresponding transactions. In this guide we will show you how to make them via `dfx` and in `Rust`. 
+For the `Rust` part you will need to setup an `ic-agent`. You can find the relevant documentation for that `crate` [here](https://crates.io/crates/ic-agent). 
+For documentation on how to install, setup and use `dfx` you can read up on the documentation [here](https://internetcomputer.org/docs/current/developer-docs/developer-tools/cli-tools/cli-reference/dfx-parent). 
+
+## Transfer tokens
+To transfer tokens you will need a few mandatory and then some optional fields.
+The mandatory fields are:
+* `to`: The account that will receive the tokens.
+* `amount`: The amount of tokens to be sent.
+The optional fields are:
+* `from_subaccount`: The subaccount of the approver.
+* `fee`: The fee paid by the sender for the transfer. If not set the default fee will have to be paid. 
+* `memo`: An arbitrary 32 bytes array to ensure transaction hash uniqueness.
```

Comment:
For consistency, we should define `memo` in the same way it is defined in the spec (as a 64-bit numerical value):  https://internetcomputer.org/docs/current/references/ledger#_transferring_tokens

Reply:
There are two different types of memos. The legacy memo is the u64 you are referring to. The recommended memo is the icrc1 memo: https://github.com/dfinity/ICRC-1/blob/main/standards/ICRC-1/README.md
which is 32 bytes.